src/gen/conditions.rs

```diff
@@ -637,7 +659,7 @@ pub fn parse_conditions(
                 } else {
                     spend.before_seconds_relative = Some(s);
                 }
-                if s <= spend.seconds_relative {
+                if Some(s) <= spend.seconds_relative {
```

Comment:
I don't think this works. I believe *any* `Some(...)` value will compare greater than `None`